.rubocop_rails.yml

```diff
@@ -0,0 +1,83 @@
+require: rubocop-rails
+
+Rails:
+  Enabled: true
+
+Rails/ActionFilter:
+  Include:
+    - decidim-*/app/controllers/**/*.rb
```

Comment:
Hi @armandfardeau,

Here we should adapt the rubocop rules to the osp-app architecture

Reply:
Thanks @Quentinchampenois. FIxed it